test/unitTests/logs/api.tests.ts

```diff
@@ -21,7 +20,7 @@ describe("logs/API", () => {
 
     before(() => {
         sandbox = sinon.createSandbox();
-        nock(DEFAULT_BREEZE_ENDPOINT)
+        nock("https://dc.services.visualstudio.com")
```

Comment:
If this constant is a dupe can we pull it from the exporter?

Reply:
Constant are not exposed in the exporter, that would require API review by Azure SDK architects